### PR TITLE
Update Content Delivery API

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update Content Delivery API:
* [x] Write shorter sentences

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Umbraco CMS v12 & 13

### CODE REVIEW

File: 12/umbraco-cms/reference/content-delivery-api/README.md [hunk 1]
```diff
@@ -578,7 +578,7 @@ GET /umbraco/delivery/api/v1/content?fetch=children:dc1f43da-49c6-4d87-b104-a586
 {% endtab %}
 
 {% tab title="filter" %}
-The `filter` query parameter allows you to specify one or more filters that must match in order for a content item to be included in the response. The API provides two built-in filters that you can use right away with the `/umbraco/delivery/api/v1/content` endpoint:
+The filter query parameter allows you to specify one or more filters. These filters must match in order for a content item to be included in the response. The API provides a few built-in filters that you can use right away with the `/umbraco/delivery/api/v2/content` endpoint:
```
Comment (alina-tincas) [inline 12/umbraco-cms/reference/content-delivery-api/README.md:581]: If I remember correctly, for v12 the api versions is v1. Will need to check this, so will confirm in a bit
```suggestion
The filter query parameter allows you to specify one or more filters. These filters must match in order for a content item to be included in the response. The API provides a few built-in filters that you can use right away with the `/umbraco/delivery/api/v1/content` endpoint:
```

Comment (alina-tincas) [reply 12/umbraco-cms/reference/content-delivery-api/README.md:581]: yeps, it is v1 for v12 so the content here is correct: 
image

You can also check on a v12 project dev env or live env cloned down by going to {{your project URL}}/umbraco/swagger 😊

**Comment by alina-tincas** [comment]
Hi @erikjanwestendorp thank you for the PR 🙌 

All looks good except changing the version for v12 content delivery to v2. I was also wondering if you would be up for making the same change from v13 to v14 as well? 😊

**Comment by erikjanwestendorp** [comment]
@alina-tincas Thanks for your review. Just updated the v12 versions and also updated v14! 😄

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Hi @erikjanwestendorp thank you for updating the v12 and v14 articles 🙌 

All looks good so am merging this in 😊

**Pull request merged**